src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -124,15 +146,25 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
 		if (!SKRIPT_COMMAND_HELP.test(sender, args))
 			return true;
 
+		boolean isReload = args[0].equalsIgnoreCase("reload");
```

Comment:
looks like this is one used once so you can just remove it and check directly instead